### DISCUSSION TITLE
[Feature request] Config

### DISCUSSION THREAD

**Discussion by Apinoita** [CorgiTaco-MC/Enhanced-Celestials#226] [Ideas]
I just downloaded your mod and I think that it is awesome. But there is one thing that bothers me and that's the announcements before the moons start. In my opinion it just breaks the immersion a little bit. That's why I'm proposing that you add a config where one could toggle the chat announcements and maybe even modify how often and what moons appear.

**Comment by CorgiTaco** [comment]
You can do this, see the wiki in regards to how

**Comment by Apinoita** [comment]
oh thanks. I didn't realize because im too used to json or toml configs. But don't the settings get reset every time you update the mod. 
Also what do write in the notification part of the lunar config? "NONE"?
